### ISSUE TITLE
Modify the /my_news_item/new

### ISSUE THREAD

**Issue by ashleygbui** [cs169/fa23-chips-10.5-14#18]
The new news article page (/representatives/[id]/my_news_item/new) has fields for Title, Link, Description, Representative, Issue, and Rating. Split this into two pages. This task is about the first page: where a user can first select the Representative and Issue they desire, and click Search

**Comment by ashleygbui** [comment]
front end done. need to create/wait on path for submit button functionality
